sql/updates/world/master/Exiles_reach.sql

```diff
@@ -0,0 +1,499 @@
+(@CGUID+11,174971,2261,10424,10639,'0',0,0,-1,0,0,3.8316922,5.474146,5.2092524,0,120,0,0,1,0,0,0,0,0,45745), -- Spar Point Advertisement
+(@CGUID+12,174971,2261,10424,10639,'0',0,0,-1,0,0,2.6647408,6.4705987,5.1801515,0,120,0,0,1,0,0,0,0,0,45745), -- Spar Point Advertisement
+(@CGUID+13,174971,2261,10424,10639,'0',0,0,-1,0,0,9.56842,4.517551,5.1502733,0,120,0,0,1,0,0,0,0,0,45745), -- Spar Point Advertisement
+(@CGUID+14,174971,2261,10424,10639,'0',0,0,-1,0,0,6.701894,6.1971374,5.194476,0,120,0,0,1,0,0,0,0,0,45745), -- Spar Point Advertisement
+(@CGUID+15,174971,2261,10424,10639,'0',0,0,-1,0,0,7.199659,-5.407894,5.216199,0,120,0,0,1,0,0,0,0,0,45745), -- Spar Point Advertisement
+(@CGUID+16,174971,2261,10424,10639,'0',0,0,-1,0,0,0.4600576,-6.25066,5.2069707,0,120,0,0,1,0,0,0,0,0,45745), -- Spar Point Advertisement
+(@CGUID+17,174971,2261,10424,10639,'0',0,0,-1,0,0,3.8914397,-6.136929,5.1904373,0,120,0,0,1,0,0,0,0,0,45745), -- Spar Point Advertisement
+(@CGUID+18,174971,2261,10424,10639,'0',0,0,-1,0,0,0.9357995,5.2422814,5.221966,0,120,0,0,1,0,0,0,0,0,45745), -- Spar Point Advertisement
+(@CGUID+19,174971,2261,10424,10639,'0',0,0,-1,0,0,6.4270716,4.5759077,5.2394466,0,120,0,0,1,0,0,0,0,0,45745), -- Spar Point Advertisement
+(@CGUID+20,174971,2261,10424,10639,'0',0,0,-1,0,0,-9.461674,-4.971889,5.456243,0,120,0,0,1,0,0,0,0,0,45745), -- Spar Point Advertisement
+(@CGUID+21,174971,2261,10424,10639,'0',0,0,-1,0,0,-7.5362535,4.9206376,5.2989116,0,120,0,0,1,0,0,0,0,0,45745), -- Spar Point Advertisement
+(@CGUID+22,174971,2261,10424,10639,'0',0,0,-1,0,0,-6.4343257,-4.8330746,5.2104034,0,120,0,0,1,0,0,0,0,0,45745), -- Spar Point Advertisement
+(@CGUID+23,174971,2261,10424,10639,'0',0,0,-1,0,0,8.676258,-4.551461,5.2418213,0,120,0,0,1,0,0,0,0,0,45745), -- Spar Point Advertisement
+(@CGUID+24,174971,2261,10424,10639,'0',0,0,-1,0,0,-3.2766402,-5.403929,5.293099,0,120,0,0,1,0,0,0,0,0,45745), -- Spar Point Advertisement
+(@CGUID+25,156403,2261,10424,10639,'0',0,0,-1,0,0,8.402318,-0.14915128,5.2594776,3.2919948,120,0,0,1,0,0,0,0,0,45745); -- aBoat
+
+DELETE FROM `spawn_group` WHERE `spawnId` BETWEEN @CGUID+0 AND @CGUID+25;
+INSERT INTO `spawn_group` (`groupId`,`spawnType`,`spawnId`) VALUES
+(129,0,@CGUID+0), -- Horde
+(129,0,@CGUID+1),
+(129,0,@CGUID+2),
+(129,0,@CGUID+3),
+(129,0,@CGUID+4),
+(129,0,@CGUID+5),
+(129,0,@CGUID+6),
+(129,0,@CGUID+7),
+(129,0,@CGUID+8),
+(129,0,@CGUID+9),
+(129,0,@CGUID+10),
+(128,0,@CGUID+11), -- Alliance
+(128,0,@CGUID+12),
+(128,0,@CGUID+13),
+(128,0,@CGUID+14),
+(128,0,@CGUID+15),
+(128,0,@CGUID+16),
+(128,0,@CGUID+17),
+(128,0,@CGUID+18),
+(128,0,@CGUID+19),
+(128,0,@CGUID+20),
+(128,0,@CGUID+21),
+(128,0,@CGUID+22),
+(128,0,@CGUID+23),
+(128,0,@CGUID+24),
+(128,0,@CGUID+25);
+
+-- *** Creature Fixes ***
+
+-- Phase updates
+UPDATE `creature` SET `PhaseId`=15286 WHERE `id` IN (166573,166824); -- Warlord Breka Grimaxe
```

Comment:
There is no PhaseShift after complete the first Quest so they are not phased.

Reply:
Changed